packages/headless/src/api/commerce/common/product.ts

```diff
@@ -81,12 +86,12 @@ export interface Product {
   additionalFields: Record<string, unknown>;
 
   /**
-   * The child results of the product, fetched through [product grouping](https://docs.coveo.com/en/l78i2152).
+   * The child products of the product, fetched through [product grouping](https://docs.coveo.com/en/l78i2152).
    */
-  childResults: Product[];
+  childProducts: ChildProduct[];
```

Comment:
@agillesCoveo @olamothe what would you think of naming this `children` instead, and naming `totalNumberOfChildrenProducts` to `totalNumberOfChildren` (on the API and in headless)? `childProducts` feels redundant to me.

Reply:
Make sense to me! Fyi I'm creating a ChildProduct model as well (to remove the children list + total number of children in the children themselft)

Reply:
Yes all good

Reply:
Applied this change in https://github.com/coveo/ui-kit/pull/3785/commits/17d9d9634f7c0f06c1bf48730a7ef0c834fc0454